docs/logs/discovery/setup.md

```diff
@@ -0,0 +1,8 @@
+title: Setup logs discovery
+description: Getting started with logs discovery
+
+To setup logs discovery you'll have to deploy the Sematext Agent in your infrastructure. Head to the [installation](/agents/sematext-agent/installation) instructions to proceed with the deployment.
```

Comment:
Maybe explain that they have to install it on all machines where files should be managed by Discovery.

Btw, that page where you would send users is pretty much useless. The thing that opens is few lines of RedHat commands that would just install the package, but without any mention of the token (essentially this wouldn't do anything for the user). Better not to send anyone there.

Reply:
Better now? @otisg 
![setup-vm-bm](https://user-images.githubusercontent.com/11174375/100234235-10fc4700-2f2b-11eb-9bd6-5c0bdeb5b534.png)

Reply:
I think so, LGTM